salt/pillar/journal-cms.sls

```diff
@@ -16,6 +16,10 @@ journal_cms:
         queue: null
         endpoint: http://10.0.2.2:4100
         topic_template: arn:aws:sns:us-east-1:512686554592:bus-%s--dev
+
+    iiif:
+        base_uri: null
```

Comment:
would still put some defaults here:
`https://dev--iiif.elifesciences.org` (doesn't exist but avoids validation and similar errors)
`/iiif` or similar